.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+---
+version: 2
+
+references:
+  git_cache_key: &git_cache_key
+    git-cache-{{ .Branch }}-{{ .Revision }}
+
+  git_save_cache: &git_save_cache
+    save_cache:
+      key: *git_cache_key
+      paths:
+        - .
+
+  git_restore_cache: &git_restore_cache
+    restore_cache:
+      key: *git_cache_key
+
+  mvn_cache_key: &mvn_cache_key
+    dependency-cache-{{ checksum "project.clj" }}
+
+  mvn_save_cache: &mvn_save_cache
+    save_cache:
+      key: *mvn_cache_key
+      paths:
+        - .m2
```

Comment:
I think this should be `/home/circleci/.m2`. I guess this is saving .m2 relative to the working directory which is /home/circle/jukebox and won't contain any jars.